tardis/io/decay.py

```diff
@@ -20,56 +21,57 @@ def __init__(self, *args, **kwargs):
     def _constructor(self):
         return IsotopeAbundances
 
-    def _update_material(self):
+    def _update_inventory(self):
         self.comp_dicts = [dict() for i in range(len(self.columns))]
         for (atomic_number, mass_number), abundances in self.iterrows():
-            nuclear_symbol = f"{nucname.name(atomic_number)}{mass_number}"
+            nuclear_symbol = f"{Z_to_elem(atomic_number)}{mass_number}"
             for i in range(len(self.columns)):
                 self.comp_dicts[i][nuclear_symbol] = abundances[i]
 
     @classmethod
-    def from_materials(cls, materials):
+    def from_inventories(cls, inventories):
         multi_index_tuples = set([])
-        for material in materials:
+        for inventory in inventories:
             multi_index_tuples.update(
-                [cls.id_to_tuple(key) for key in material.keys()]
+                [cls.id_to_tuple(key) for key in inventory.contents.keys()]
             )
 
         index = pd.MultiIndex.from_tuples(
             multi_index_tuples, names=["atomic_number", "mass_number"]
         )
 
         abundances = pd.DataFrame(
-            data=0.0, index=index, columns=range(len(materials))
+            data=0.0, index=index, columns=range(len(inventories))
         )
 
-        for i, material in enumerate(materials):
-            for key, value in material.items():
-                abundances.loc[cls.id_to_tuple(key), i] = value
+        for i, inventory in enumerate(inventories):
+            for nuclide, abundance in inventory.mass_fractions():
+                abundances.loc[cls.id_to_tuple(nuclide), i] = abundance
 
         return cls(abundances)
 
     @staticmethod
     def id_to_tuple(atomic_id):
-        return nucname.znum(atomic_id), nucname.anum(atomic_id)
+        nuclide = Nuclide(atomic_id)
+        return nuclide.Z, nuclide.A
 
-    def to_materials(self):
+    def to_inventories(self):
         """
-        Convert DataFrame to a list of materials interpreting the MultiIndex as
+        Convert DataFrame to a list of inventories interpreting the MultiIndex as
         atomic_number and mass_number
 
         Returns
         -------
         list
-            list of pyne Materials
+            list of radioactivedecay Inventories
         """
 
         comp_dicts = [dict() for i in range(len(self.columns))]
         for (atomic_number, mass_number), abundances in self.iterrows():
-            nuclear_symbol = f"{nucname.name(atomic_number):s}{mass_number:d}"
+            nuclear_symbol = f"{Z_to_elem(atomic_number)}{mass_number}"
             for i in range(len(self.columns)):
                 comp_dicts[i][nuclear_symbol] = abundances[i]
-        return [material.Material(comp_dict) for comp_dict in comp_dicts]
+        return [Inventory(comp_dict, "kg") for comp_dict in comp_dicts]
```

Comment:
We use cgs units in tardis. Can you make this grams?

Reply:
Since we use mass fractions in TARDIS, converting from grams to kgs should be unnecessary

Reply:
Turns out `radioactivedecay` can handle grams - the newest commit has these changes